files/zlux/config/zluxserver.json

```diff
@@ -30,6 +30,14 @@
       "ipAddresses": ["127.0.0.1"],
       //to be a replacement for zssPort
       "port": 8542
+    },
+     "jwt": {
+      "enabled": true,
+      "fallback": false,
+      "key": {
+        "token": "TOKEN.NAME",
+        "label": "KEY_LABEL"
```

Comment:
Can you add some detail to the ticket about this.... are token & label supposed to be substituted by $PKCS11_TOKEN_NAME and $PKCS11_TOKEN_LABEL?
If so, no changes to this file are needed as this behavior can occur dynamically via reading the env vars at startup.

Reply:
Basically, something like:
```
if [ -z $PKCS11_TOKEN_NAME ]
then
export ZWED_agent_jwt_key_token=$PKCS11_TOKEN_NAME
fi
```
within https://github.com/zowe/zlux-app-server/blob/master/bin/convert-env.sh

Reply:
Your assumption is correct, those values should be substituted by environment variables. I will update the doc I am working on as well, and revert this change